administrator/components/com_admin/sql/updates/postgresql/5.3.0-2025-01-23.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE "#__content_types"
+SET "field_mappings" = jsonb_set("field_mappings"::jsonb, '{common, core_created_user_id}' , '"created_by"', true),
+    "field_mappings" = jsonb_set("field_mappings"::jsonb, '{common, core_created_by_alias}' , '"created_by_alias"', true)
+WHERE "type_alias" = 'com_content';

```

Comment:
```suggestion
UPDATE "#__content_types"
SET "field_mappings" = jsonb_set("field_mappings"::jsonb, '{common, core_created_user_id}' , '"created_by"', true)
WHERE "type_alias" = 'com_content';

UPDATE "#__content_types"
SET "field_mappings" = jsonb_set("field_mappings"::jsonb, '{common, core_created_by_alias}' , '"created_by_alias"', true)
WHERE "type_alias" = 'com_content';
```
We should use separate update statements for each modification to avoid an SQL error `multiple assignments to same column "field_mappings"`.